Dockerfile-arm

```diff
@@ -6,26 +6,29 @@ RUN unzip /tmp/terraform_*.zip -d /tmp
 # it's offical so i'm using it + alpine so damn small
 FROM python:3.7.4-alpine3.10
 
-# copy the codebase
-COPY . /www
-RUN chmod +x /www/terraformize_runner.py
+#exposing the port
+EXPOSE 80
 
-# copy terraform binary and make it executable
-COPY --from=terraform_installer /tmp/terraform /usr/bin/terraform
-RUN chmod +x /usr/bin/terraform
+#set python to be unbuffered
+ENV PYTHONUNBUFFERED=1
 
 # install required packages - requires build-base due to gevent GCC complier requirements
-RUN apk add --no-cache build-base libffi-dev
-RUN pip install -r /www/requirements.txt
+RUN apk add --no-cache \
+	build-base \
+	libffi-dev
+
+# copy terraform binary
+COPY --from=terraform /bin/terraform /usr/local/bin/terraform
```

Comment:
--from=terraform should be --from=terraform_installer or line 2 will need to be changed so `AS terraform`

Reply:
DOH